### DISCUSSION TITLE
How to ensure third-party services and integrations are properly configured after converting a personal account to an organization?

### DISCUSSION THREAD

**Discussion by holdehn** [supabase/supabase#13430] [Questions]
Hello GitHub and Supabase Community!

I'm in urgent need of assistance. I recently converted my personal GitHub account into an organization, and now I cannot access my Supabase project at all. I'm worried that it might have been deleted or lost during the conversion process.

Can anyone provide guidance on how to quickly recover my Supabase project or help me verify if it still exists? Are there any common issues or known solutions related to Supabase integration after converting a personal GitHub account to an organization? I'm really concerned about my project and need to resolve this issue as soon as possible.

If anyone has experienced a similar issue or has any advice, tips, or resources, your help would be greatly appreciated. Thank you in advance for your support during this critical situation.

**Comment by GaryAustin1** [comment]
Probably best to contact support.  Even if your email changes in github you have to contact them currently to get connected again to your dashboard.
